CRITICAL - 2023-01-11 08:14:34 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 226.
 1 SYSTEMPATH\CodeIgniter.php(226): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(190): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(55): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2023-01-11 08:16:52 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 226.
 1 SYSTEMPATH\CodeIgniter.php(226): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(190): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(55): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2023-01-11 08:17:04 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 226.
 1 SYSTEMPATH\CodeIgniter.php(226): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(190): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(55): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2023-01-11 08:17:32 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 226.
 1 SYSTEMPATH\CodeIgniter.php(226): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(190): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 ROOTPATH\spark(83): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2023-01-11 08:18:55 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 226.
 1 SYSTEMPATH\CodeIgniter.php(226): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(190): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 ROOTPATH\spark(83): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2023-01-11 08:22:48 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 226.
 1 SYSTEMPATH\CodeIgniter.php(226): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(190): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 ROOTPATH\spark(83): CodeIgniter\CodeIgniter->initialize()
ERROR - 2023-01-11 09:00:17 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\crud-generator\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\crud-generator\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#4 C:\xampp\htdocs\crud-generator\app\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
#5 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(934): App\Controllers\Home->index()
#6 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\crud-generator\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\crud-generator\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\crud-generator\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\crud-generator\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#3 C:\xampp\htdocs\crud-generator\app\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
#4 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(934): App\Controllers\Home->index()
#5 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\crud-generator\public\index.php(67): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\crud-generator\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2023-01-11 09:00:17 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
 3 APPPATH\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-01-11 09:00:38 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\crud-generator\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\crud-generator\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#4 C:\xampp\htdocs\crud-generator\app\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
#5 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(934): App\Controllers\Home->index()
#6 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\crud-generator\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\crud-generator\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\crud-generator\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\crud-generator\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#3 C:\xampp\htdocs\crud-generator\app\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
#4 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(934): App\Controllers\Home->index()
#5 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\crud-generator\public\index.php(67): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\crud-generator\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2023-01-11 09:00:38 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
 3 APPPATH\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-01-11 09:00:54 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\crud-generator\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\crud-generator\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#4 C:\xampp\htdocs\crud-generator\app\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
#5 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(934): App\Controllers\Home->index()
#6 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\crud-generator\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\crud-generator\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\crud-generator\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\crud-generator\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#3 C:\xampp\htdocs\crud-generator\app\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
#4 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(934): App\Controllers\Home->index()
#5 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\crud-generator\public\index.php(67): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\crud-generator\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2023-01-11 09:00:54 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
 3 APPPATH\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-01-11 09:04:39 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\crud-generator\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\crud-generator\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#4 C:\xampp\htdocs\crud-generator\app\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
#5 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(934): App\Controllers\Home->index()
#6 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\crud-generator\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\crud-generator\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\crud-generator\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\crud-generator\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#3 C:\xampp\htdocs\crud-generator\app\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
#4 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(934): App\Controllers\Home->index()
#5 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\crud-generator\public\index.php(67): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\crud-generator\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2023-01-11 09:04:39 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
 3 APPPATH\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-01-11 09:07:37 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\crud-generator\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\crud-generator\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#4 C:\xampp\htdocs\crud-generator\app\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
#5 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(934): App\Controllers\Home->index()
#6 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\crud-generator\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\crud-generator\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\crud-generator\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\crud-generator\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#3 C:\xampp\htdocs\crud-generator\app\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
#4 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(934): App\Controllers\Home->index()
#5 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\crud-generator\public\index.php(67): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\crud-generator\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2023-01-11 09:07:37 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
 3 APPPATH\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-01-11 09:07:56 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\crud-generator\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\crud-generator\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#4 C:\xampp\htdocs\crud-generator\app\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
#5 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(934): App\Controllers\Home->index()
#6 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\crud-generator\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\crud-generator\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\crud-generator\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\crud-generator\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#3 C:\xampp\htdocs\crud-generator\app\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
#4 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(934): App\Controllers\Home->index()
#5 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\crud-generator\public\index.php(67): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\crud-generator\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2023-01-11 09:07:56 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
 3 APPPATH\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-01-11 09:08:11 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\crud-generator\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\crud-generator\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#4 C:\xampp\htdocs\crud-generator\app\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
#5 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(934): App\Controllers\Home->index()
#6 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\crud-generator\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\crud-generator\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\crud-generator\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\crud-generator\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#3 C:\xampp\htdocs\crud-generator\app\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
#4 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(934): App\Controllers\Home->index()
#5 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\crud-generator\public\index.php(67): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\crud-generator\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2023-01-11 09:08:11 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
 3 APPPATH\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-01-11 09:08:14 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\crud-generator\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\crud-generator\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#4 C:\xampp\htdocs\crud-generator\app\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
#5 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(934): App\Controllers\Home->index()
#6 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\crud-generator\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\crud-generator\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\crud-generator\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\crud-generator\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#3 C:\xampp\htdocs\crud-generator\app\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
#4 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(934): App\Controllers\Home->index()
#5 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\crud-generator\public\index.php(67): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\crud-generator\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2023-01-11 09:08:14 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
 3 APPPATH\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-01-11 09:09:44 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\crud-generator\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\crud-generator\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#4 C:\xampp\htdocs\crud-generator\app\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
#5 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(934): App\Controllers\Home->index()
#6 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\crud-generator\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\crud-generator\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\crud-generator\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\crud-generator\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#3 C:\xampp\htdocs\crud-generator\app\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
#4 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(934): App\Controllers\Home->index()
#5 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\crud-generator\public\index.php(67): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\crud-generator\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2023-01-11 09:09:44 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
 3 APPPATH\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-01-11 09:10:23 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\crud-generator\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\crud-generator\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#4 C:\xampp\htdocs\crud-generator\app\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
#5 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(934): App\Controllers\Home->index()
#6 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\crud-generator\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\crud-generator\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\crud-generator\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\crud-generator\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#3 C:\xampp\htdocs\crud-generator\app\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
#4 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(934): App\Controllers\Home->index()
#5 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\crud-generator\public\index.php(67): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\crud-generator\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2023-01-11 09:10:23 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
 3 APPPATH\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-01-11 09:12:06 --> db_crud_generator is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 SYSTEMPATH\Database\Config.php(107): CodeIgniter\Database\Config::connect('db_crud_generator')
 2 APPPATH\Controllers\Home.php(9): CodeIgniter\Database\Config::forge('db_crud_generator')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-01-11 09:13:01 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\crud-generator\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\crud-generator\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#4 C:\xampp\htdocs\crud-generator\app\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
#5 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(934): App\Controllers\Home->index()
#6 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\crud-generator\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\crud-generator\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\crud-generator\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\crud-generator\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#3 C:\xampp\htdocs\crud-generator\app\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
#4 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(934): App\Controllers\Home->index()
#5 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\crud-generator\public\index.php(67): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\crud-generator\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2023-01-11 09:13:01 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
 3 APPPATH\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-01-11 09:13:06 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\crud-generator\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\crud-generator\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#4 C:\xampp\htdocs\crud-generator\app\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
#5 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(934): App\Controllers\Home->index()
#6 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\crud-generator\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\crud-generator\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\crud-generator\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\crud-generator\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#3 C:\xampp\htdocs\crud-generator\app\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
#4 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(934): App\Controllers\Home->index()
#5 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\crud-generator\public\index.php(67): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\crud-generator\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2023-01-11 09:13:06 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
 3 APPPATH\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-01-11 09:13:29 --> Call to a member function get() on null
in APPPATH\Controllers\Home.php on line 13.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-01-11 09:13:43 --> Call to a member function get() on null
in APPPATH\Controllers\Home.php on line 13.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-01-11 09:16:27 --> Call to a member function get() on null
in APPPATH\Controllers\Home.php on line 13.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-01-11 09:16:37 --> Invalid file: "welcome_messagesad.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('welcome_messagesad.php')
 2 SYSTEMPATH\Common.php(1177): CodeIgniter\View\View->render('welcome_messagesad', [], true)
 3 APPPATH\Controllers\Home.php(22): view('welcome_messagesad')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-01-11 09:16:46 --> Invalid file: "welcome_messagesad.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('welcome_messagesad.php')
 2 SYSTEMPATH\Common.php(1177): CodeIgniter\View\View->render('welcome_messagesad', [], true)
 3 APPPATH\Controllers\Home.php(22): view('welcome_messagesad')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-01-11 09:17:19 --> Invalid file: "welcome_messagesad.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('welcome_messagesad.php')
 2 SYSTEMPATH\Common.php(1177): CodeIgniter\View\View->render('welcome_messagesad', [], true)
 3 APPPATH\Controllers\Home.php(22): view('welcome_messagesad')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-01-11 09:17:33 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\crud-generator\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\crud-generator\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#4 C:\xampp\htdocs\crud-generator\app\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
#5 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(934): App\Controllers\Home->index()
#6 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\crud-generator\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\crud-generator\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\crud-generator\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\crud-generator\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#3 C:\xampp\htdocs\crud-generator\app\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
#4 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(934): App\Controllers\Home->index()
#5 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\crud-generator\public\index.php(67): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\crud-generator\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2023-01-11 09:17:33 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
 3 APPPATH\Controllers\Home.php(9): CodeIgniter\Database\Config::forge()
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-01-11 09:18:21 --> Field information is required.
in SYSTEMPATH\Database\Forge.php on line 534.
 1 APPPATH\Controllers\Home.php(11): CodeIgniter\Database\Forge->createTable('table_name')
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-01-11 15:02:18 --> Undefined property: App\Controllers\Home::$db
in APPPATH\Controllers\Home.php on line 23.
 1 APPPATH\Controllers\Home.php(23): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined property: App\\Controllers\\Home::$db', 'APPPATH\\Controllers\\Home.php', 23, [])
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-01-11 15:02:52 --> Undefined property: App\Controllers\Home::$db
in APPPATH\Controllers\Home.php on line 23.
 1 APPPATH\Controllers\Home.php(23): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined property: App\\Controllers\\Home::$db', 'APPPATH\\Controllers\\Home.php', 23, [])
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-01-11 15:03:24 --> Call to undefined method CodeIgniter\Database\MySQLi\Connection::list_fields()
in APPPATH\Controllers\Home.php on line 24.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-01-11 15:15:13 --> mysqli_sql_exception: Table 'db_crud_generator.table_name' doesn't exist in C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php(295): mysqli->query('SHOW INDEX FROM...', 0)
#1 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SHOW INDEX FROM...')
#2 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SHOW INDEX FROM...')
#3 C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php(444): CodeIgniter\Database\BaseConnection->query('SHOW INDEX FROM...')
#4 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(1532): CodeIgniter\Database\MySQLi\Connection->_indexData('`table_name`')
#5 C:\xampp\htdocs\crud-generator\app\Controllers\Home.php(29): CodeIgniter\Database\BaseConnection->getIndexData('table_name')
#6 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(934): App\Controllers\Home->index()
#7 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\crud-generator\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\crud-generator\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-01-11 15:15:13 --> Table 'db_crud_generator.table_name' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(444): CodeIgniter\Database\BaseConnection->query('SHOW INDEX FROM `table_name`')
 2 SYSTEMPATH\Database\BaseConnection.php(1532): CodeIgniter\Database\MySQLi\Connection->_indexData('`table_name`')
 3 APPPATH\Controllers\Home.php(29): CodeIgniter\Database\BaseConnection->getIndexData('table_name')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-01-11 15:15:16 --> mysqli_sql_exception: Table 'db_crud_generator.table_name' doesn't exist in C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php(295): mysqli->query('SHOW INDEX FROM...', 0)
#1 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SHOW INDEX FROM...')
#2 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SHOW INDEX FROM...')
#3 C:\xampp\htdocs\crud-generator\system\Database\MySQLi\Connection.php(444): CodeIgniter\Database\BaseConnection->query('SHOW INDEX FROM...')
#4 C:\xampp\htdocs\crud-generator\system\Database\BaseConnection.php(1532): CodeIgniter\Database\MySQLi\Connection->_indexData('`table_name`')
#5 C:\xampp\htdocs\crud-generator\app\Controllers\Home.php(29): CodeIgniter\Database\BaseConnection->getIndexData('table_name')
#6 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(934): App\Controllers\Home->index()
#7 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\crud-generator\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\crud-generator\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\crud-generator\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-01-11 15:15:16 --> Table 'db_crud_generator.table_name' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(444): CodeIgniter\Database\BaseConnection->query('SHOW INDEX FROM `table_name`')
 2 SYSTEMPATH\Database\BaseConnection.php(1532): CodeIgniter\Database\MySQLi\Connection->_indexData('`table_name`')
 3 APPPATH\Controllers\Home.php(29): CodeIgniter\Database\BaseConnection->getIndexData('table_name')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-01-11 22:58:42 --> Undefined variable: table
in APPPATH\Helpers\custom_helper.php on line 8.
 1 APPPATH\Helpers\custom_helper.php(8): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: table', 'APPPATH\\Helpers\\custom_helper.php', 8, [...])
 2 APPPATH\Views\builder.php(19): check_have_pk('pl_brands')
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\builder.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1177): CodeIgniter\View\View->render('builder', [], true)
 6 APPPATH\Controllers\Builder.php(14): view('builder', [...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Builder->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Builder))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
